modules/javafx.controls/src/main/java/javafx/scene/control/skin/TextAreaSkin.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2011, 2022, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2011, 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
minor: year should be 2024.
We update copyright years in bulk before releases/new year, so in general one can avoid changing copyright year along with fix.